### DISCUSSION TITLE
No git folder, Readme & gitignore inside the practice folder

### DISCUSSION THREAD

**Discussion by Amrita987** [Jay-study-nildana/FrontEndForStudents#39] [Coding Questions]
Hello Jay Sir,

I am a current batch students Full stack internship program. I am following the steps and trying to catch up but while i already got the authentication pop-up, meanwhile when i was waiting for the recording to go ahead i missed the pop up and it takes me the master branch.

i tried more than 10 times but the folder is empty and i am stuck with that particular thing. I get the pop-up with regular login to GitHub login and can't login too, not the one with the continue with the browser which i got earlier?

I hope you understand my question.

Thank you!
Amrita

**Comment by Jay-study-nildana** [comment]
@Amrita987 

I don’t actually understand your question. 

1. GitHub pop up login only has to be done one time. After that, that step is not required. But you are saying you cannot login. For this, I strongly recommend you watch this video on my YouTube channel. You can find the videos at this link : https://www.youtube.com/playlist?list=PL7SHzFbuSdW0LWKYOOMPVzVVCa_BtnuZb 

1. Tried 10 times. But what did you try? You have not mentioned what you tried. So, again, no clue. Once you watch the videos on my channel, you will see that, first you create the repository on github.com, then ‘clone’ it on your computer. After cloning, you will get a folder, and inside it will have .git and read me files. Again, watch the videos. 

Finally, if you are in Bangalore, I conduct, free github workshops. If possible, attend and learn. 

<img width="540" alt="image" src="https://github.com/Jay-study-nildana/FrontEndForStudents/assets/14297315/01255b96-ed01-4a1a-8d7f-9bc548339d1b">

Hope this helps. Happy coding. Remember to google my name “Vijayasimha BR” and follow me on social media. You will learn a lot about software development simply by following my online videos, posts and other things. 

Thanks

Jay

**Comment by Jay-study-nildana** [comment]
@CodeByZaid

(related issue, https://github.com/Jay-study-nildana/FrontEndForStudents/discussions/41)

I have given the answer above, same question, from another student. 

Thanks. If you have more questions, reply on this thread. 

Jay
